### DISCUSSION TITLE
Can i integrate filament with a my own package ?

### DISCUSSION THREAD

**Discussion by lucacalcaterra** [filamentphp/filament#2004] [Help] [answered]
I would to know if it's possible to integrate filament with a my own package to split pages and resources.

For example create aadmin pages only for Blog in a package and another package for Users, splitting them easily...

Thanks !

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/admin/plugins

**Reply by lucacalcaterra** [reply]
Thanks, i've started to study the spatie package for this purpose but now try with this dedicated method. Thanks !

**Reply by lucacalcaterra** [reply]
> https://filamentphp.com/docs/2.x/admin/plugins

I made some progress creating simple package but i cannot understand one thing:  migrations sould be published or must run from package ? I used Spatie package tools template but seems migrations not work without publish them

Thanks

**Reply by danharrin** [reply]
Just checked, it looks like Spatie's package tools just adds the migrations as publishable. But, you can use the traditional Laravel method instead - https://laravel.com/docs/9.x/packages#migrations
